aas_core_meta/v3.py

```diff
@@ -24,6 +24,10 @@
 
 * :constraintref:`AASd-012`
 
+:constraintref:`AASd-116`: In this constraint the string ``globalAssetId`` is checked case-sensitively,
```

Comment:
```suggestion
* :constraintref:`AASd-116`: In this constraint the string ``globalAssetId`` is checked case-sensitively,
```

@s-heppner I think this is also an item in the preceding bullet list ("language understanding")?